### PR TITLE
[fix] force utf-8 encoding in windows

### PR DESCRIPTION
**Pull request by marmoure**
On windows the response was not shown correctly as seen in this image
image

this PR fixes the encoding issue

### CODE REVIEW

**Review by adamretter — APPROVED** [review]
LGTM

File: src/main/java/org/humanistika/oxygen/tei/completer/response/impl/JSONTransformer.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
 import static java.nio.charset.StandardCharsets.UTF_8;
 
+import java.nio.charset.StandardCharsets;
```
Comment (adamretter) [inline src/main/java/org/humanistika/oxygen/tei/completer/response/impl/JSONTransformer.java]: This import is unneeded isn't it?

**Review by adamretter — CHANGES_REQUESTED** [review]
One question...

File: src/main/java/org/humanistika/oxygen/tei/completer/response/impl/JSONTransformer.java (hunk 1 above)
Comment (marmoure) [reply src/main/java/org/humanistika/oxygen/tei/completer/response/impl/JSONTransformer.java]: Correct the IDE added it when i was going over the options

**Review by adamretter — APPROVED** [review]
LGTM

**Pull request merged**